src/commands/install.yml

```diff
@@ -68,22 +46,179 @@ parameters:
       https://github.com/yarnpkg/yarn/releases
 
 steps:
-  - install-node:
-      version: <<parameters.node-version>>
-      alpine-version: <<parameters.node-alpine-version>>
-      install-dir: <<parameters.node-install-dir>>
-      debug: <<parameters.debug>>
+  - run:
+      name: Installing NodeJS
+      command: |
+        # PLATFORM CHECK: mac vs. alpine vs. other linux
+
+        SYS_ENV_PLATFORM=linux
+        if uname -a | grep Darwin; then
+          SYS_ENV_PLATFORM=darwin
+        elif cat /etc/issue | grep Alpine; then
+          SYS_ENV_PLATFORM=alpine
+        fi
+
+        # FUNCTIONS
+
+        nvm_check () {
+          if [[ -n "$NVM_DIR" ]]; then
```

Comment:
I think it might be good to test that `NVM_DIR` is set after installing nvm, I did a quick test and it doesn't seem to be set. Perhaps the standard nvm installer doesn't write to $BASH_ENV

Reply:
I've opted to remove the NVM checks for now given the orbs main functions are aimed at our cimgs. But if we run into issues it's no problem to revisit it. (and as you mentioned, may not have been working)